tempesta_fw/t/functional/flacky_net/test_sockets.py

```diff
@@ -0,0 +1,147 @@
+"""All sockets must be closed after Tempesta shutdown"""
```

Comment:
Well, I don't care about Guido's point of view on how a beauty code must looks like. I just verified that tabs work find for Python3:

```python
$ cat tail_recursion.py 
#!/usr/bin/python3

def tail_factorial(total, n):
	if not n:
		return total
	return tail_factorial(total * n, n - 1)

def tail_factorial2(total, n):
	if not n:
		return total
	return (lambda : tail_factorial2(total * n, n - 1))

def tail_rec(total, n):
	g = tail_factorial2(total, n)
	while hasattr(g, '__call__'):
		total = total * tail_factorial

print(tail_factorial(1, 10))
$ ./tail_recursion.py 
3628800
```

The code used tabs 8 characters in width. It seems Pylint allows to specify what should it use for identation https://stackoverflow.com/questions/25303726/how-to-fix-pylint-wrong-hanging-indentation-and-pep8-e121